index.html

```diff
@@ -1848,7 +1848,7 @@ <h2 class="text-white text-center text-3xl font-bold pb-4">
         class="md:col-start-5 lg:col-start-6 col-start-2 ml-auto mr-auto"
         style="align-content: center; display: inherit"
       >
-        <a href="https://horovod.ai" target="_blank">
+        <a href="https://docs.flyte.org/projects/cookbook/en/latest/auto/integrations/kubernetes/kfmpi/index.html" target="_blank">
```

Comment:
Can you link the text to https://docs.flyte.org/projects/cookbook/en/latest/auto/case_studies/ml_training/spark_horovod/index.html?